lua/starfall/libs_sh/quaternion.lua

```diff
@@ -44,7 +44,7 @@ local function wrap(tbl)
 	return setmetatable(tbl, quat_meta)
 end
 
-local function unpack(tbl)
+local function qunpack(tbl)
 	return self[1], self[2], self[3], self[4]
```

Comment:
Lots of your local functions use 'self' in them and should be the argument instead

Reply:
Outdated, it was already fixed in https://github.com/thegrb93/StarfallEx/pull/953/commits/78a068c93d67fcf0dc158ba3c1b9a1c5c96678e1